test/integration/actions.test.js

```diff
@@ -112,14 +116,14 @@ test('create, get and delete with parameters an action', t => {
   }).catch(errors)
 })
 
-test('invoke action with fully-qualified name', t => {
+tempTest('invoke action with fully-qualified name', t => {
```

Comment:
catalog can be deploy using postdeploy in ansible.
alarm, kafka, and cloudant can't be deploy right now. but things like `/whisk.system/utils/echo` should be able to deploy